ui/ceval/src/view.ts

```diff
@@ -379,6 +414,7 @@ function renderPvMoves(pos: Position, pv: Uci[]): VNode[] {
         {
           key,
           attrs: {
+            'data-move-index': i,
```

Comment:
does that work with `.dataset['moveIndex']`?

Reply:
Yeah, I didn't know either, but the dashes are converted to camel case when accessing the data.